src/internal/easee/auth.go

```diff
@@ -45,6 +80,10 @@ func (a *authenticator) Login(userName, password string) error {
 		return fmt.Errorf("failed to save credentials in storage: %w", err)
 	}
 
+	err = a.notificationManager.Event(&notification.Event{EventName: notificationEaseeStatusOnline})
```

Comment:
I'm not sure if we need a push notification here. The user will do it via a mobile app, so the result of an operation will be known immediately.

Reply:
The intention was rather to inform user on a successful connection to Easee in the context of previous disconnection message, but I removed it. We have a similar message in the playground.